### DISCUSSION TITLE
Drawer inside dashboards

### DISCUSSION THREAD

**Discussion by nicolastakashi** [grafana/grafana#52507] [Dashboards, Panels & Visualizations]
Hey Team!

I would like to suggest a feature using [drawers](https://github.com/grafana/grafana/blob/23956557d8c6a119b7de5be5c42024e29634d002/packages/grafana-ui/src/components/Drawer/Drawer.mdx) inside dashboards.

When we're building more complex visualizations, it's really easy to decrease the UX because we start to have lots of rows and panels in only one dashboard.

The Drawer component enables us to display additional information without hiding the main page content, it could be awesome, especially during correlation with information from a context different from the main dashboard.

I can describe how I imagine this working.

1 - Create a dashboard with tons of rows and panels
2 - Add a new Drawer (it could be a new link option)
3 - The Drawer allows you add new panels inside the drawer using the variables that you have on the main dashboard and also add its own variables.

WDYT?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
